website/docs/docs/deploy/about-state.md

```diff
@@ -26,9 +26,26 @@ Together, these two features enable ["slim CI"](best-practices#run-only-modified
 
 State and defer can be set by environment variables as well as CLI flags:
 
+<VersionBlock lastVersion="1.4">
+
 - `--state` or `DBT_ARTIFACT_STATE_PATH`: file path
 - `--defer` or `DBT_DEFER_TO_STATE`: boolean
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+- `--state` or `DBT_STATE`: file path
+- `--defer` or `DBT_DEFER`: boolean
+
+:::warning Syntax deprecated
+
+The original syntax for state (`DBT_ARTIFACT_STATE_PATH`) and defer (`DBT_DEFER_TO_STATE`) have been deprecated in dbt v1.5. Backward compatibility is supported in this version but will be removed in an as-of-yet-undetermined future release.
```

Comment:
this is not a blocker and merely a suggestion on switching to active voice. it may be an instance where passive is ok per style guide.

```suggestion
The original syntax for state (`DBT_ARTIFACT_STATE_PATH`) and defer (`DBT_DEFER_TO_STATE`) have been deprecated in dbt v1.5. dbt supports backward compatibility with the old syntax, but we will remove it in a future release that has not yet been determined.
```